crawl-ref/source/dat/des/branches/crypt.des

```diff
@@ -1463,6 +1486,884 @@ cG.MGM.GcG.1.GcG.1.GcG.1.Gc
+KMASK:   p- = no_monster_gen
+KMASK:   p- = no_trap_gen
+KMASK:   p- = no_item_gen
+: dgn.delayed_decay(_G, 'p', 'human corpse')
+MAP
+xxxxxxxxxxxxxx
+xxx``xp-xq!xxx
+xxx``xPPxQQxxx
+x``........``x
+x``........``x
+xxx..GxxG..xxx
+x``..xxxx..``x
+x``..xxxx..``x
+xxx..GxxG..xxx
+Rrr........``x
+Rrr........``x
+xxxssx``x``xxx
+xxxssx``x``xxx
+xxxSSxxxxxxxxx
+ENDMAP
+
+NAME:   nicolae_crypt_fracryptal
+TAGS:   transparent
+DEPTH:  Crypt
+NSUBST: Q = @ / x
+SUBST:  - = %0...., x = xxxxxxx., % = %%%*, 0 = 0009, * = ***|, 9 = 8889
+MAP
+xxxxxxxQxxxxxxxQxxxxxxxQxxxxxxx
+x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x
+x.....x.x.....x.x.....x.x.....x
+x-x.x-x.x-x.x-x.x-x.x-x.x-x.x-x
+xxx.xxx.xxx.xxx.xxx.xxx.xxx.xxx
+Q-...........-x.x-...........-Q
+xxx.xxx.xxx.xxx.xxx.xxx.xxx.xxx
+x-x.x-x.x-x.x-x.x-x.x-x.x-x.x-x
+x.....x.x.....x.x.....x.x.....x
+x-x-x-x.x-x-x-x.x-x-x-x.x-x-x-x
+xxxxxxx.xxxxxxx.xxxxxxx.xxxxxxx
+Q-...........................-Q
+xxxxxxx.xxxxxxx.xxxxxxx.xxxxxxx
+x-x-x-x.x-x-x-x.x-x-x-x.x-x-x-x
+x.....x.x.....x.x.....x.x.....x
+x-x.x-x.x-x.x-x.x-x.x-x.x-x.x-x
+xxx.xxx.xxx.xxx.xxx.xxx.xxx.xxx
+Q-...........-x.x-...........-Q
+xxx.xxx.xxx.xxx.xxx.xxx.xxx.xxx
+x-x.x-x.x-x.x-x.x-x.x-x.x-x.x-x
+x.....x.x.....x.x.....x.x.....x
+x-x-x-x-x-x-x-x-x-x-x-x-x-x-x-x
+xxxxxxxQxxxxxxxQxxxxxxxQxxxxxxx
+ENDMAP
+
+NAME:   nicolae_crypt_golgotha
+TAGS:   transparent
+DEPTH:  Crypt
+KMONS:  S = curse skull
+KMONS:  s = flying skull
+SUBST:  S = SSsss, s = sss.
+MAP
+     .........
+     ..x.x.x..
+     .x.x.x.x.
+     .........
+     .s.xxx.s.
+.........x.........
+..x.s.xx...xx.s.x..
+.x....xx.S.xx....x.
+..x.x.........x.x..
+.x..xx.S.S.S.xx..x.
+..x.x.........x.x..
+.x....xx.S.xx....x.
+..x.s.xx...xx.s.x..
+.........x.........
+     .s.xxx.s.
+     .........
+     .x.x.x.x.
+     ..x.x.x..
+     .........
+ENDMAP
+
+NAME:  nicolae_crypt_grand_chamber
+TAGS:  transparent
+DEPTH: Crypt
+KMONS: E = guardian mummy / death scarab w:5 / ushabti w:5 / nothing w:30
+KMONS: P = mummy priest
+SUBST: * = ***|
+MAP
+xxxxxxxxxxxxxxxxxxxxxx
+xxxx*x.x.x.xxxxxxxxxxx
+xxx*..E.E.E.xxxxxxxxxx
+xx*x.x.x.x.x.x.xxxxxxx
+x*..E.E.E.E.E.E.xxxxxx
+xx.x.....x.x.x.x.x.x.x
+x*..E.P.E.E.E.E......+
+xx.x.....x.x.x.x.x.x.x
+x*..E.E.E.E.E.E.xxxxxx
+xx*x.x.x.x.x.x.xxxxxxx
+xxx*..E.E.E.xxxxxxxxxx
+xxxx*x.x.x.xxxxxxxxxxx
+xxxxxxxxxxxxxxxxxxxxxx
+ENDMAP
+
+NAME:    nicolae_crypt_grave_offerings
+TAGS:    transparent
+DEPTH:   Crypt
+KITEM:   a = $ / % / * / | w:5
+KMONS:   a = 0 / 9 w:5 / nothing w:30
+SHUFFLE: DEFH, JK
+SUBST:   D : @, E : @x, F : @xx, H : @xxx, J : x.., K : ., L : x..
+MAP
+   xxxEEExxx
+  xx.......xx
+ xx.........xx
+xx...JK.KJ...xx
+x...LxxGxxL...x
+D....xaaa+....F
+D....xaaa+....F
+x...LxxGxxL...x
+xx...JK.KJ...xx
+ xx.........xx
+  xx.......xx
+   xxxHHHxxx
+ENDMAP
+
+NAME:   nicolae_crypt_heckumenical_temple
+TAGS:   transparent
+DEPTH:  Crypt
+KMONS:  9 = 9
+KFEAT:  9 = Y
+KFEAT:  O = altar_beogh
+KFEAT:  D = altar_dithmenos
+KFEAT:  K = altar_kikubaaqudgha
+KFEAT:  L = altar_lugonu
```

Comment:
I know this is supposed to be a list of all the Evil gods, but the two branch gods feel pretty weird to see in a non-D/Depths vault that doesn't feature much of their spawns or decoration; orc high priests haven't had Animate Dead in eight years, so Beogh looks lost in Crypt, while Lugonu has a bunch of undead in the Abyss but rather explicitly doesn't get along with the other gods much to appear in this temple. I'd replace the Beogh and Lugonu altars with some mix of a room full of sacrificed corpses, statues, or the blood fountain moved out of the center, and just let a box of boxes with Dith / Kiku / Makhleb / Yred already look notable, imo.